src/main/java/sirius/biz/storage/layer2/BlobDispatcher.java

```diff
@@ -181,12 +181,26 @@ public DispatchDecision dispatch(WebContext request) throws Exception {
         Values uriParts = Values.of(uri.split("/"));
         String type = uriParts.at(0).asString();
         if (Strings.areEqual(type, PHYSICAL_DELIVERY) && uriParts.length() == 5) {
```

Comment:
What is the difference between a length of 5 or 6? Constants could also be helpful here.

Reply:
Also please read my comment above.
The main difference is the length of the URL or in otherwords the number of parts of which the URL consists of